woocommerce/woocommerce-admin
# Add onboarding profile wizard step 1

Fixes #1884

Adds step 1 content and styling changes.

This isn't entirely complete, but I think we can get some early feedback and/or merge and follow-up on a few issues here:
* Before merge this needs a rebase to match the new styles in #2264.
* We're still waiting on form component and button style changes, so no styling changes have been made to the toggles, checkboxes, or buttons in this PR.
* I'm planning on tackling the API calls in a follow-up PR, so currently the actions don't respond.

### Screenshots
<img width="1195" alt="Screen Shot 2019-05-21 at 3 55 32 PM" src="https://user-images.githubusercontent.com/10561050/58078264-e1a3a600-7be0-11e9-93a5-d14d357be3cc.png">

### Detailed test instructions:

1.  Set `const profileWizardComplete = false;` in `dashboard/index.js`.
2.  Go to the dashboard at `/wp-admin/admin.php?page=wc-admin#/`
3.  Note the styling matches the designs (pending #2264 updates).
4. Make sure the checkbox (toggle on < 782px) are in sync.
